src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines/MapAction_ExplicitSource_SimpleReturn_Snapshot.generated.txt

```diff
@@ -172,6 +168,346 @@ namespace Microsoft.AspNetCore.Http.Generated
                 createRequestDelegate);
         }
 
+        [InterceptsLocation(@"TestMapActions.cs", 26, 5)]
+        internal static RouteHandlerBuilder MapGet1(
```

Comment:
Each of these handlers has the same signature but different parameter sources and names and now generates different interceptor methods.